cg/store/crud/read.py

```diff
@@ -1736,11 +1737,16 @@ def get_all_pools_to_deliver(self) -> list[Pool]:
         )
         return records.all()
 
-    def get_orders(self, limit: int | None = None) -> list[Order]:
+    def get_orders(self, orders_request: OrdersRequest | None = None) -> list[Order]:
         """Returns a list of entries in the table Order."""
         records: Query = self._get_query(table=Order)
-        if limit is not None:
-            records = records.limit(limit)
+        if orders_request:
```

Comment:
Extract this to a separate utils method called `parse_orders_request` let it return a `OrdersRequest` pydantic model containing the limit and workflow